23.2.2 Added (#1)
friedcucumber/maui-htmledit-customize-toolbar: README.md

# DevExpress .NET MAUI HTML Edit - Replicate a Built-In Toolbar

> The DevExpress **HTML Edit** control for .NET MAUI includes the built-in [DXToolbar Control](https://docs.devexpress.com/MAUI/404604/dialogs-menu-and-navigation/toolbar/toolbar-overview?v=23.2) that you can use to manipulate the displayed content. This repository replicates the **HTML Edit**'s built-in toolbar in XAML including the [SafeKeyboardAreaView](https://docs.devexpress.com/MAUI/DevExpress.Maui.Core.SafeKeyboardAreaView?v=23.2&) class. You can use this class to achieve the following:

Reviewer: ```suggestion
The DevExpress **HTML Edit** control for .NET MAUI includes a built-in [DXToolbar Control](https://docs.devexpress.com/MAUI/404604/dialogs-menu-and-navigation/toolbar/toolbar-overview?v=23.2). This example replicates the **HTML Edit**'s built-in toolbar in XAML and uses a [SafeKeyboardAreaView](https://docs.devexpress.com/MAUI/DevExpress.Maui.Core.SafeKeyboardAreaView?v=23.2&) container. This class helps you achieve the following:
```

> * Decrease the height of HTML Edit when you open the device keyboard and keep the toolbar visible. In other words, you avoid overlap between the keyboard and the toolbar.

Reviewer 2: ```suggestion
* Decreases the height of the HTML Edit when you open the device keyboard and keep the toolbar visible. In other words, avoids an overlap between the keyboard and the toolbar.
```

> * Decrease the height of HTML Edit when you open the device keyboard and keeps the toolbar visible. It allows to avoid the device keyboard and the toolbar overlap.

Reviewer: ```suggestion
* Decrease the height of HTML Edit when you open the device keyboard and keep the toolbar visible. In other words, you avoid overlap between the keyboard and the toolbar.
```

> * Display custom content in the keyboard area to add more space for UI elements.

Reviewer 2: ```suggestion
* Displays custom content in the keyboard area to add more space for UI elements.
```
